### PR TITLE
lseg blog post

### PR DESCRIPTION
**Pull request by mario-guerra**

### CODE REVIEW

**Comment by mario-guerra** [comment]
@allenjzhang @bterlson plz approve.

**Review by allenjzhang — APPROVED** [review]

File: blog/2024-10-04-typespec-at-lseg/blog.md
Comment (timotheeguerin) [inline blog/2024-10-04-typespec-at-lseg/blog.md:1]: that should move to `website/src/content/blog` for the new website

File: blog/2024-10-04-typespec-at-lseg/blog.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+### Accessibility
+
+The compact nature of TypeSpec, combined with its familiar JavaScript-style syntax, allows even non-technical stakeholders to participate in the API review and design process. This inclusiveness ensures that APIs are designed with a broader perspective, incorporating insights from various roles within the organization, and ultimately leading to more user-centric and well-rounded API designs.
+
+### High-Level SDK Generation
+
+TypeSpec provides a pluggable generation architecture via "emitters," which can be used to provide customized output. LSEG partnered with the Industry Solutions Engineering (ISE) group at Microsoft to build a customized emitter specific to LSEG's requirements and needs.
```
[the remark this answers is not in the document]
Comment (mario-guerra) [reply blog/2024-10-04-typespec-at-lseg/blog.md:55]: That was language chosen by the Industry Solutions Engineering team that did the work.

File: blog/2024-10-04-typespec-at-lseg/blog.md
Comment (mario-guerra) [reply blog/2024-10-04-typespec-at-lseg/blog.md:1]: Content moved.

**Review by timotheeguerin — APPROVED** [review]

**Pull request merged**